starfish/intensity_table.py

```diff
@@ -88,10 +90,12 @@ def empty_intensity_table(
             empty IntensityTable
 
         """
-        cls._verify_spot_attributes(spot_attributes)
+        if not isinstance(spot_attributes, SpotAttributes):
```

Comment:
Unclear why we validate the data type here where we don't really elsewhere.

Reply:
Added check to `from_spot_data.` 👍

Reply:
Actually, I think you can forgo all this verification.  That is what type annotations is supposed to do for you.  The only place we don't have type annotation coverage is the notebooks, and it's likely it'll bomb out anyway if it's the wrong type.